CONTRIBUTING.md

```diff
@@ -45,6 +46,15 @@ It is mandatory to provide end-to-end (e2e) tests for new scaler. For more infor
 check the [test documentation](./tests/README.md). Those tests are run nightly on our
 [CI system](https://github.com/kedacore/keda/actions?query=workflow%3A%22nightly+e2e+test%22).
 
+## Changelog
+
+Every change should bve added to our changelog under `Unreleased` which is located in `CHANGELOG.md`. This helps us keep track of all changes in a given release.
+
+Here are some guidelines to follow:
+- Always use `General: ` or `<Scaler Name>: ` as a prefix and sort them alphabetically
+  - General changes, however, should always be at the top
+- Entries should always follow the `<Scaler Name / General>: <Description> (#<ID>)` where `<ID>` is preferrably the ID of an issue, otherwise a PR is OK.
```

Comment:
could the issue have more noise than the PR?
Both are ok for me

Reply:
I'm not sure what you mean? Is the suggestion to add a link to the PR template or?

Reply:
oh sorry, yes, I meant that I'm not sure if linking the issue instead of the PR is better or the issue could have a lot of things already linked to it (like other issues or PRs in this repo or others).

Reply:
But linking the issue or the PR is good for me

Reply:
> oh sorry, yes, I meant that I'm not sure if linking the issue instead of the PR is better or the issue could have a lot of things already linked to it (like other issues or PRs in this repo or others).

The issue should always be the starting point IMO as it goes beyond just one PR and includes other things such as related items and doc PRs.